spec/api/v1/external_clearstream_list_request_spec.rb

```diff
@@ -0,0 +1,73 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require '/home/lex/Clients/Concur/Projects/stub_api/spec/helpers.rb'
+
+RSpec.configure do |c|
+  c.include Helpers
+end
+
+RSpec.describe 'External list requests to Clearstream', type: :request do
+  it 'get all lists' do
+    uri = URI('https://api.getclearstream.com/v1/lists')
+    # Send request to fake clearstream server (redirected to FackClearstream server)
+    response = JSON.parse(Net::HTTP.get(uri))
+    expect(response['all']['data'].first['id']).to eq 25_041
+  end
+
+  it 'get one list' do
```

Comment:
```suggestion
  it 'gets one list' do
```

Reply:
Let's clean up the tests in the next feature branch, okay?